### PR TITLE
Fix show course supervisor

### PR DESCRIPTION
**Pull request by hienbx94**
I have created pull request. Please check it.

### CODE REVIEW

**Comment by lannp** [comment]
OK

**Comment by vietanhbkaaa** [comment]
OK

File: app/views/rails_admin/main/show_course_subject.html.erb [hunk 1]
```diff
@@ -41,7 +41,14 @@
             </strong></td>
             <td>
               <% @tasks.each do |task| %>
-                <li><%= task.name %></li>
+                <tr>
+                  <td><%= task.decorate.get_name %></td>
+                  <td>
+                    <%= link_to t("admin.actions.remove_member.title"),
+                      rails_admin.delete_path(Task, task), method: :delete,
```
Comment (hellovietnam93) [inline app/views/rails_admin/main/show_course_subject.html.erb]: sử dụng ajax

**Comment by hellovietnam93** [comment]
OK

**Pull request closed without merging**